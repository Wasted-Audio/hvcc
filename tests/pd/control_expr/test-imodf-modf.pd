#N canvas 827 239 734 565 12;
#X obj 40 22 loadbang;
#X obj 40 49 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 40 230 print;
#X msg 78 137 -2;
#X msg 40 137 0.5;
#X obj 40 194 expr imodf($f1);
#X obj 155 230 print;
#X obj 155 194 expr modf($f1);
#X obj 40 80 t b b;
#X connect 0 0 1 0;
#X connect 1 0 8 0;
#X connect 3 0 5 0;
#X connect 3 0 7 0;
#X connect 4 0 5 0;
#X connect 4 0 7 0;
#X connect 5 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 4 0;
#X connect 8 1 3 0;
