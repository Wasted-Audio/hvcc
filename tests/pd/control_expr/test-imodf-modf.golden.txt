[@ 0.000] print: -2
[@ 0.000] print: -0
[@ 0.000] print: 0
[@ 0.000] print: 0.5
